app/javascript/Redux/ConfigureStore.js

```diff
@@ -0,0 +1,20 @@
+import { combineReducers, configureStore } from "@reduxjs/toolkit"
+import getDetailsReducer from "./hotelDetailsReducer"
+import HomePageReducer from "./HomePageReducer"
```

Comment:
- [ ] Your imports still cannot be resolved by the Javascript parser and thus Webpack is unable to compile them for new users/installs (of course, it works for you because you already have the build files as explained in the last review):

![image](https://user-images.githubusercontent.com/57421823/201386406-24ed132e-88f0-4220-a0fd-bbddf1c61784.png)

![image](https://user-images.githubusercontent.com/57421823/201386479-9f3e8c22-b9b6-4be2-b8df-c6fbd8d308f2.png)

![image](https://user-images.githubusercontent.com/57421823/201386535-273c5a5d-4abb-450c-a3a7-ee79e54f9a19.png)

![image](https://user-images.githubusercontent.com/57421823/201386596-02f102db-4ba1-4d36-801c-767125ca00ad.png)

## Explanation

I'm really sorry I wasn't able to explain myself clearly last time.  Look:

![image](https://user-images.githubusercontent.com/57421823/201387060-84fc547b-fddc-4aaf-a221-e83200e1cd04.png)

Look very closely at the above screenshot ☝️ 

"hotelDetailsReducer" is NOT the same as "HotelDetailsReducer"
"HomePageReducer" is NOT the same as "HomepageReducer"

Please, notice the difference in capitalization.  The name of the files should be exactly the same, including capitalization in order for the imports to work.  Javascript and programming in general are case-sensitive.

## Solution

Please, kindly change those 2 imports to:

```javascript
import getDetailsReducer from "./HotelDetailsReducer"
import HomePageReducer from "./HomepageReducer"
```

... and they will work:

![image](https://user-images.githubusercontent.com/57421823/201388729-f98e9162-bbde-428b-9117-255018241db1.png)

### I repeat: It works for you because you already have the build files from past builds, but it doesn't work for new people.
### Please, fix those 2 imports because the final evaluator and potential employers will surely run into this problem too when trying to run your app.
### If you need more help, I'm available please don't hesitate to reach out.